tests/e2e/pkg/tests/tests.go

```diff
@@ -0,0 +1,568 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/banzaicloud/koperator/tests/e2e/pkg/common"
+	"github.com/banzaicloud/koperator/tests/e2e/pkg/common/config"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/dsl/decorators"
+	"github.com/spf13/viper"
+	"golang.org/x/exp/maps"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//	[]testCase
+//		|
+// Classifier ==> TestPool <-- []Test <-- K8sCluster <-- ClusterInfo
+// 		^						 ^
+//		|						 |
+// 	K8sClusterPool 			   testCase
```

Comment:
This multi-line comment might be a bit out-of-format

Reply:
Interesting. For me in VS studio it looks good.

Reply:
@bartam1 would you mind fixing the format of this multi-line comment?

Reply:
Huhhh, yea, I also just checked it locally and the format is fine with Goland as well. Probably it is indeed something fishy with GitHub's UI